src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVCFConstants.java

```diff
@@ -162,7 +162,8 @@ their names (or descriptions) depend on some threshold.  Those filters are not i
     public final static String N_RATIO_FILTER_NAME =                           "n_ratio";
     public final static String CHIMERIC_ORIGINAL_ALIGNMENT_FILTER_NAME =       "numt_chimera"; //mitochondria
     public final static String ALLELE_FRACTION_FILTER_NAME =                   "low_allele_frac";
-    public static final String POTENTIAL_POLYMORPHIC_NUMT_FILTER_NAME =        "numt_novel";
+    public static final String POSSIBLE_NUMT_FILTER_NAME =                     "possible_numt";
+    public static final String LOW_HET_FILTER_NAME =                            "low_het";
```

Comment:
I try to keep every M2 filter name at least 8 characters and less than 16.  That way the filtering stats file and, with luck, the VCF, looks better when viewed in a terminal.